python/shotgun_fields/image_widget.py

```diff
@@ -305,8 +305,7 @@ def _display_value(self, value):
             In delegate mode, this value can also be an existing ``QPixmap``
             object.
         """
-
-        if isinstance(value, QtGui.QPixmap):
+        if type(value).__name__ == "QPixmap":
```

Comment:
Can we use the tk-core patcher class instead?

Reply:
In this file, `QtGui` is imported `from sgtk.platform.qt`, Can you elaborate more?

Reply:
> In PySide6, value is type <class 'PySide6.QtGui.QPixmap'> instead.

Which qt "interface" are you using? Is this to start porting to QT6 from QT4 interface?

Reply:
@staceyoue I think we're still using QT4 interface. This change is because there's a class mismatch when it's running on PySide6, detected by tk-multi-demo.

Reply:
Ok yes, we are still using QT4 interface then but with PySide6 under the hood.

The reason this gives you an error is because for PySide6 with QT4 interface we patch the QPixmap class [here](https://github.com/shotgunsoftware/tk-core/blob/67cb1bccbe3b0c5743ddbeb312e42409366c3b47/python/tank/util/pyside6_patcher.py#L88-L111) but some how the tk-multi-demo is creating the QPixmap object not through this patched class, instead it is creating it through PySide6 original QPixmap class, which is why you see `<class 'PySide6.QtGui.QPixmap'>` when it should be something like `<class 'sgtk.platform.qt.QtGui.QPixmap'>`.

Reply:
Now I used a new util method introduced on https://github.com/shotgunsoftware/tk-core/pull/975